### ISSUE TITLE
[unit-tests] FeedbackLink a11y test is flaky

### ISSUE THREAD

**Issue by JammingBen** [owncloud/web#8244]
The unit test `has no accessibility violations` of the `FeedbackLink` component is flaky, sometime it fails: `Error: connect ECONNREFUSED 127.0.0.1:80`

**Comment by pascalwengerter** [comment]
Closing as fixed via #8245 (I guess?)

**Comment by pascalwengerter** [comment]
Closing as "can't find the respective test anymore"...
